src/worker.js

```diff
@@ -7,20 +7,17 @@ import BatchManager from './utils/BatchManager';
 
 let closing = false;
 
-export default (app, config, onServer, workerId) => {
-  let server;
-
-  // ===== Middleware =========================================================
+export const attachMiddleware = (app, config) => {
```

Comment:
Ok, updated it to me more like CJS, to work without `default`.

Let me know if you ok with clustering being not default.

Thanks.